_includes/bottomnav.html

```diff
@@ -14,12 +14,7 @@
       <div class="col-6 col-md">
         <h5>Features</h5>
         <ul class="list-unstyled text-small">
-          <li><a class="text-muted" href="#">Cool stuff</a></li>
-          <li><a class="text-muted" href="#">Random feature</a></li>
-          <li><a class="text-muted" href="#">Team feature</a></li>
-          <li><a class="text-muted" href="#">Dive into the lake</a></li>
-          <li><a class="text-muted" href="#">Another one</a></li>
-          <li><a class="text-muted" href="#">Last time</a></li>
+          <li><a class="text-muted" href="divemasters.html">Our Team</a></li>
```

Comment:
change divemasters.html to /divemasters so then when the user navigates 
- they get a clean experience on the target URL
- navigation works from anywhere in the site